### ISSUE TITLE
Did RoBERTa longer trained?

### ISSUE THREAD

**Issue by bayartsogt-ya** [uds-lsv/bert-stable-fine-tuning#1]
Thank you for great work and paper!

[1] I wonder if you guys trained RoBERTa longer than 3 epochs?
[2] Is there any log other than one included in the paper?

**Comment by mmarius** [comment]
Hi, thank you! 

For the RoBERTa results that are presented in the paper we trained for just 3 epochs. But if you fine-tune it for longer and take care of using warmup and bias correction you will see improved stability. This finding is consistent across all models we tried (BERT, RoBERTa, ALBERT).

**Comment by bayartsogt-ya** [comment]
Thank you for your quick reply.
Are you guys having an intention to share the distribution graph of results of the other models (other than just BERT)?

**Comment by mmarius** [comment]
We have them in our paper when training for 3 epochs (Fig. 5).

**Comment by bayartsogt-ya** [comment]
@mmarius 
Thank you so much!
Closing the issue.
